filesystems/blob/src/test/scala/cromwell/filesystems/blob/BlobFileSystemContractSpec.scala

```diff
@@ -0,0 +1,46 @@
+package cromwell.filesystems.blob
+
+import au.com.dius.pact.consumer.PactTestExecutionContext
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import pact4s.scalatest.RequestResponsePactForger
+import au.com.dius.pact.core.model.RequestResponsePact
+import au.com.dius.pact.consumer.ConsumerPactBuilder
+import io.circe.Json
+import io.circe.syntax._
+import pact4s.circe.implicits._
+
+class BlobFileSystemContractSpec extends AnyFlatSpec with Matchers with RequestResponsePactForger {
```

Comment:
Does this currently get run when we run Cromwell unit tests? Does the pact file get automatically generated?

Reply:
I believe so, but of course without any actual tests it won't run anything. The pact file will get generated automatically if all tests pass in this file. It will not however get published to the pact broker automatically is my understanding.

Reply:
Sounds good! Could you leave a comment here describing the current state of this test so we'll know where to pick up when we come back to this work?